README.md

```diff
@@ -300,6 +300,114 @@ config.disabledTools = disabledTools;
 config.customHeaders = {'headerName': 'headerValue'};
 PdftronFlutter.openDocument(_document, config: config);
 ```
+### PdftronFlutter.importAnnotations(String)
+Imports XFDF string to current document.
+
+```dart
+
+var xfdfCommand = '<?xml version="1.0" encoding="UTF-8"?>\n<xfdf xmlns="http://ns.adobe.com/xfdf/" xml:space="preserve">\n\t<annots>\n\t\t<circle style="solid" width="5" color="#E44234" opacity="1" creationdate="D:20190729202215Z" flags="print" date="D:20190729202215Z" page="0" rect="138.824,653.226,236.28,725.159" title="" /></annots>\n\t<pages>\n\t\t<defmtx matrix="1.333333,0.000000,0.000000,-1.333333,0.000000,1056.000000" />\n\t</pages>\n\t<pdf-info version="2" xmlns="http://www.pdftron.com/pdfinfo" />\n</xfdf>';
+PdftronFlutter.importAnnotations(xfdfCommand);
+```
+
+### PdftronFlutter.exportAnnotations(List<`PTAnnot`>)
+To extract XFDF from the current document. If `annotationList` is null, export all annotations from the document; Else export the valid ones specified.
+
+For more details about `PTAnnot`, please check `lib/options.dart` file.
```

Comment:
Wondering why the type is `PTAnnot`? The `PT` prefix is an Obj-C thing due to lack of namespaces. Unless there is a reason to prefix the class, we should just stick with `Annot` in Dart.